### PR TITLE
Create a Quick Start Java Sample

### PR DESCRIPTION
**Pull request by MeganRoach**
Created a Java web app which utilizes the Immersive Reader SDK. This acts as a sample to demonstrate to users how to include the Immersive Reader SDK into a Java web app. Included detailed instructions for how to get the sample up and running in the ReadMe file.

### CODE REVIEW

File: samples/quickstart-java/src/main/webapp/index.jsp [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <div class="container body-content">
+        <html>
+<head>
+    <meta charset='utf-8'>
+    <title>Immersive Reader Example: Document</title>
+    <script type='text/javascript' src='http://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-0.0.1.js'></script>
```
Comment (metanMSFT) [inline samples/quickstart-java/src/main/webapp/index.jsp]: Can you change this to immersive-reader-sdk.0.0.1.js? This is the new path

File: samples/quickstart-java/src/main/webapp/index.jsp [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <div class="navbar-collapse collapse">
+                <ul class="nav navbar-nav">
+                    <li><a href="/">Home</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <div class="container body-content">
+        <html>
+<head>
```
Comment (metanMSFT) [inline samples/quickstart-java/src/main/webapp/index.jsp]: Remove html tag, head tag, meta tag, title tag. Move the script tag under line 7.

File: samples/quickstart-java/src/main/webapp/index.jsp [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    <meta charset='utf-8'>
+    <title>Immersive Reader Example: Document</title>
+    <script type='text/javascript' src='http://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-0.0.1.js'></script>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+</head>
```
Comment (metanMSFT) [inline samples/quickstart-java/src/main/webapp/index.jsp]: Remove head tag, body tag

File: samples/quickstart-java/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+            ImmersiveReader.launchAsync(token, data, options)
+                .then(() => {
+                    console.log('success');
+                }, (error) => {
+                    console.log('error! ' + error);
+                });
+        }
+    </script>
+</body>
```
Comment (metanMSFT) [inline samples/quickstart-java/src/main/webapp/index.jsp]: Remove body tag and html tag

File: samples/quickstart-java/src/main/webapp/index.jsp (hunk 1 above)
Comment (MeganRoach) [reply samples/quickstart-java/src/main/webapp/index.jsp]: Yes

**Pull request merged**